lib/internal/http2/compat.js

```diff
@@ -439,20 +459,37 @@ class Http2ServerResponse extends Stream {
     stream.respond(headers);
   }
 
+  createPushResponse(headers, callback) {
+    var stream = this[kStream];
+    if (stream === undefined) {
+      throw new Error('HTTP/2 Stream has been closed');
+    }
+    try {
+      stream.pushStream(headers, {}, function(stream, headers, options) {
+        var response = new Http2ServerResponse(stream);
+        callback(null, response);
+      });
+    } catch (error) {
+      callback(error);
+    }
```

Comment:
Habit from projects with other coding styles. Changed it up to simply throw instead.